src/main/steps/sequence/irretrievable-breakdown/content.ts

```diff
@@ -2,9 +2,9 @@ import { TranslationFn } from '../../../app/controller/GetController';
 import { FormContent } from '../../../app/form/Form';
 import { isFieldFilledIn } from '../../../app/form/validation';
 
-export const generateContent = (title: string): TranslationFn => ({ isDivorce }) => {
+export const generateContent: TranslationFn = ({ isDivorce }) => {
   const en = {
-    title: isDivorce ? title : 'Has your relationship irretrievably broken down (it cannot be saved)?',
+    title: 'Has your relationship irretrievably broken down (it cannot be saved)?',
```

Comment:
You'll need a `isDivorce` title here

Reply:
I believe it was the same

Reply:
It was not the same. I cleaned up the logic in there because it was pretty repetitive